include/inicpp.h

```diff
@@ -419,6 +419,7 @@ namespace ini
         char esc_ = '\\';
         std::vector<std::string> commentPrefixes_ = { "#" , ";" };
         bool multiLineValues_ = false;
+        bool exceptionWhenDuplicateField_ = false;
```

Comment:
Generally I prefer positive formulation, so here you could  rename the member to something like `overwriteDuplicateFields_ = true`